### ISSUE TITLE
Idle E1.31 refresh rate is too slow

### ISSUE THREAD

**Issue by marcusbirkin** [OpenLightingProject/ola#1288]
The idle refresh rate of the E1.31 plug in has been observed, via wireshark, as being between a packet every 5-10seconds.
E1.31 6.9.1 states that a period of 2.5seconds or greater, should be treated as a data loss condition.

Observed in OLA git commit e4bd7acb072973a90fb574a36003c9eb15b76b33
Built on Debian Linux

**Comment by nomis52** [comment]
The transmit rate depends on the source of the DMX data. If the source doesn't generate any new DMX nothing will be sent over E1.31. That is there are no 'keep alives'.

**Comment by marcusbirkin** [comment]
The source in this case was the Olad web interface.
The source was still active (aka tab was open), but the data was static.

What you are saying is that it's the Ola sources responsibly to ensure refresh rate compliance and not the Ola output plugin?
This seems like a bad idea, as different standards will require different minimum refresh rates. 

In the case of E1.31, not maintaing the minimum refresh rate will trigger a data loss condition. This will most likely result in a DBO on stage.

**Comment by nomis52** [comment]
>What you are saying is that it's the Ola sources responsibly to ensure refresh rate compliance and not the Ola output plugin?

Correct. OLA is a protocol translator, it does not generate "phantom" DMX frames (with a few exceptions)

This was an design choice very early on - users wanted a one-in-one-out component so they could control the rate of DMX transmission. We know this is different from pretty much every other "gateway" device on the market.

The web based DMX console *should* be sending a frame every 1s. If it's not we have a bug.

**Comment by peternewman** [comment]
Hi @marcusbirkin ,

Where you using the new or old web UI, which page (faders or keypad?) and what browser/OS?
